.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,30 +1,38 @@
 ---
 name: 🛠 Bug report
 about: Create a report to help us improve
-title: "[Bug Report] Good bug title tells us about precise symptom, not about the root cause."
+title: "Good bug title tells us about precise symptom, not about the root cause."
```

Comment:
I removed this label because we have the label, this allows longer title and reduced noise IMO